backend/docker-compose.yml

```diff
@@ -4,7 +4,6 @@ name: dokedu
 services:
   db:
     image: postgres:16-alpine
-    restart: always
```

Comment:
Out of curiosity, why did you remove the auto restart from the postgres container?

Reply:
The restart flag is a pretty important one for server or production deployments. If the host gets restarted, docker updated or the container crashes, it will automatically get restarted.

However, this also means that this container starts every time I turn my PC on, and blocks the default 5432 port, which is pretty annoying when I am working on other projects that also need port 5432

Reply:
Gotcha! Thanks for the explanation and removal. I used to stop the containers when switching between projects, but I also rarely restart my PC